Clients/Xamarin.Interactive.Client.Windows/AgentSessionWindow.xaml

```diff
@@ -180,12 +180,13 @@
             TabStripPlacement="Top"
             SelectedIndex="{Binding ViewModel.SelectedPane, Mode=TwoWay}"
             SelectionChanged="HandleSelectedPaneChanged"
-            MouseDown="HandleTabControlMouseDown">
+            MouseDown="HandleTabControlMouseDown"
+            AutomationProperties.AutomationId="SessionWindowTabControl">
             <TabControl.Resources>
                 <Style BasedOn="{StaticResource PrimaryContentTabPanel}" TargetType="{x:Type TabPanel}"/>
                 <Style BasedOn="{StaticResource PrimaryContentTabItem}" TargetType="{x:Type TabItem}"/>
             </TabControl.Resources>
-            <TabItem Header="{Binding ViewModel.IsInspector, Converter={StaticResource replHeaderConverter}}">
+            <TabItem Header="{Binding ViewModel.IsInspector, Converter={StaticResource replHeaderConverter}}" AutomationProperties.AutomationId="WorkbooksTabItem">
```

Comment:
If we're putting an ID here, we should do the same for the next `TabItem`, while we're at it.